packages/dart_frog/example/main.dart

```diff
@@ -0,0 +1,6 @@
+import 'package:dart_frog/dart_frog.dart';
+
+// Learn more at https://github.com/VeryGoodOpenSource/dart_frog
```

Comment:
Isn't this a form self-referencing, considering that this is the example of a package in the same repo? I dont mind, but I normally don't see this in examples of multi repos either, unless it is the README.md